packages/shared/src/utils.ts

```diff
@@ -81,9 +90,9 @@ export function createArrayFromView<T, U extends TypedArray | T[]>(
   const { data } = view;
 
   const array = ndarray(
-    // eslint-disable-next-line @typescript-eslint/prefer-ts-expect-error
-    // @ts-ignore (`data.constructor` resolves to `Function`)
-    (isTypedArray(data) ? new data.constructor(view.size) : []) as U,
+    (isTypedArray(data)
+      ? new (data.constructor as TypedArrayConstructor)(view.size)
+      : []) as U,
```

Comment:
I managed to avoid the `ts-ignore` with another assertion.

Reply:
Meh. No choice :shrug: